profiles/centosamazon.go

```diff
@@ -119,7 +119,7 @@ func NewCentosAmazonCluster(name string) *cluster.Cluster {
 							{
 								IngressFromPort: "0",
 								IngressToPort:   "65535",
-								IngressSource:   "10.0.100.0/24",
```

Comment:
I have a couple of questions regarding this - 

- I see the Kube nodes are created with this subnet - 10.0.100.0/24. How does this rule help traffic from nodes to master, or we don't need traffic from nodes to master ?

- What is listening on port 65535 in master ?

Reply:
@prateekgogia the nodes are created within the specified CIDR - these mentioned above are firewall rules specifying the allowed source and range. Master is in 10.0.0.0/24 and allows ingress from Node which is 10.0.100.0/24 - based on the rule, and the other way around.
There is nothing listening in 65535, this is just the port range (from 0 to 65535 - due to dynamic ports you can't list them all) and -1 is the allowed protocols (TCP UDP ICMP).